templates/authors/author_page.html

```diff
@@ -0,0 +1,77 @@
+{% extends 'base_page.html' %}
+
+{% load static %}
+{% load wagtailcore_tags %}
+{% load wagtailimages_tags %}
+{% block breadcrumb %}{% endblock %}
+{% block content %}
+    {% include 'includes/breadcrumb.html' %}
+    <div class="author-intro">
+        <div class="container">
+            <div class="row">
+                <div class="col-sm-12 col-md-3 order-md-2">
```

Comment:
Ideally we need to move away from the bootstrap grid towards the TNA one.

The equivalent for this would be:

```html
<div class="tna-container">
    <div class="tna-column tna-column--1-4 tna-column--full-tiny tna-column--order-3 tna-column--order-1-tiny">
        <!-- IMAGE -->
    </div>
    <div class="tna-column tna-column--3-4 tna-column--full-tiny tna-column--order-2">
        <!-- INFO -->
    </div>
</div>
```

Reply:
_Some_ of the documentation for it is here: https://nationalarchives.github.io/design-system/components/grid/